doc/upgrading.txt

```diff
@@ -1,8 +1,18 @@
 Upgrading
 =========
+Migrating from 9.x to 10.0
+-------------------------
+Python 3.5 or newer is required.
+
+The Python Beautiful Soup package is now required. A C compiler is not needed
+for building.
+
+The loginuserfield and loginpasswordfield entries in the authentication section
+of linkcheckerrc are now matched to a login form case-sensitively.
+
 Migrating from 8.x to 9.0
 -------------------------
-The Python requests module is now required.
+The Python requests package is now required.
 
 Several checks have been moved to plugins (see below).
 Plugins have to be enabled in the configuration file.
```

Comment:
A couple of lines down from this I see this text which is news to me:

> The following commandline and configuration options have been deprecated
> and do not have any effect:

@cjmayo, I remember you had a PR removing those options?  I objected because I thought t might cause pain to users if we remove working functionality, and that declaring it deprecated was not a good enough reason to do that.  I didn't realize it wasn't actually **working** functionality.  I retract my objection.

Reply:
New PR with just the options commit:  #413.